components/motor/fake/motor.go

```diff
@@ -97,8 +93,24 @@ type Motor struct {
 	Logger logging.Logger
 }
 
+// NewMotor creates a new fake motor.
+func NewMotor(ctx context.Context, deps resource.Dependencies, conf resource.Config, logger logging.Logger) (motor.Motor, error) {
+	logger.Error("in new motor")
+	m := &Motor{
+		Named:  conf.ResourceName().AsNamed(),
+		Logger: logger,
+		OpMgr:  operation.NewSingleOperationManager(),
+	}
+	if err := m.Reconfigure(ctx, deps, conf); err != nil {
+		return nil, err
+	}
+	logger.Errorf("new motor = %v", m)
```

Comment:
This doesn't look like an error-level log to me. Any chance this was here only for debugging the PR and should be removed now?

Reply:
yeah just a mistake to leave all these error statements from debugging my changes